exporter/otlp/README.md

```diff
@@ -60,6 +61,13 @@ tracer.in_span('foo') do |span|
     pp child_span
   end
 end
+
+# Immediately export all spans that have not yet been exported.
+# This method should only be called in cases where it is absolutely
+# necessary, such as when using some FaaS providers that may suspend
+# the process after an invocation, but before the `Processor` exports
+# the completed spans.
+tracer_provider.force_flush
```

Comment:
```suggestion
tracer_provider.shutdown
```
This will attempt to gracefully shutdown the SDK, including flushing the BatchSpanProcessor

Reply:
Thank you for taking a look! I didn't realize that method 👍 
PTAL https://github.com/open-telemetry/opentelemetry-ruby/pull/1373/commits/67938eee227ee8881ef7c5788b4c001ded41d04c

Reply:
Thank you for taking a look, I didn't realize that method 👍 
PTAL
https://github.com/open-telemetry/opentelemetry-ruby/pull/1373/commits/3fedc2346a2109c7a5158870a721624f28581694